appcenter-data/Data.js

```diff
@@ -46,9 +69,11 @@ Data.WriteOptions = class {
 };
 
 function convertTimestampToDate(result) {
-    // Create a new `Date` object from timestamp as milliseconds
+    // Create a new `Date` object from timestamp as milliseconds.
     result.lastUpdatedDate = new Date(result.lastUpdatedDate);
     return result;
 }
 
+this.read();
```

Comment:
What is this?

Reply:
This is committed by accident. It's removed.